hw/xbox/nv2a/nv2a_psh.c

```diff
@@ -965,12 +953,14 @@ QString *psh_translate(const PshState state)
 {
     int i;
     struct PixelShader ps;
-    memset(&ps, 0, sizeof(ps));
+    struct InputInfo blank;
```

Comment:
nit: How is this even used? Looks like it only serves as a throw-away?
If so, I'd rather pass a `NULL` and let the function deal with it internally.